scripts/first-run.py

```diff
@@ -46,8 +46,9 @@ def lose(msg, e=None):
 except Exception as e:
     lose("Could not use git to find my own repository URL", e)
 
-m = re.match(r"https://github.com/(.*)/(.*)/?", str(my_repo_url))
```

Comment:
This works and is easy to understand, which is great! But there's a simpler way to do i:

```
m = re.match(r"(?:https://github.com/|git@github.com:)(.*)/(.*)/?", str(my_repo_url))
```

Reply:
Looks good, thanks! I pushed a new commit that uses the simpler way.